tests/integration/util/standalone.rs

```diff
@@ -33,13 +33,18 @@ impl StandaloneCluster {
         let conn_str = c.conn_string().unwrap();
         let username = c.username().unwrap();
         let password = c.password().unwrap();
-        let features = StandaloneCluster::get_supported_features(
-            conn_str.clone(),
-            bucket.clone(),
-            username.clone(),
-            password.clone(),
-        )
-        .await;
+        let features = if conn_str.contains("couchbases") {
```

Comment:
This seems weird to me, doesn't this mean that we can't test with SSL against a non-capella cluster?